relayer/relays/beacon/header/syncer/syncer.go

```diff
@@ -59,12 +52,20 @@ type finalizedUpdateContainer struct {
 }
 
 func (s *Syncer) GetCheckpoint() (scale.BeaconCheckpoint, error) {
-	checkpoint, err := s.Client.GetFinalizedCheckpoint()
+	retries := 5
+	bootstrap, err := s.getCheckpoint()
 	if err != nil {
-		return scale.BeaconCheckpoint{}, fmt.Errorf("get finalized checkpoint: %w", err)
+		for retries > 0 {
+			retries = retries - 1
+			bootstrap, err = s.getCheckpoint()
+			if err != nil {
+				log.WithError(err).Info("retry bootstrap, sleeping")
+				time.Sleep(10 * time.Second)
+				continue
+			}
+			break
```

Comment:
Just curious is the retry necessary here?

Reply:
I got intermittent failures from Lodestar where a bootstrap payload was not available for the provided finalized checkpoint, so I added a retry, instead of having to retry the whole e2e script. :)